test/spec/adUnits_spec.js

```diff
@@ -41,7 +41,7 @@ describe('Publisher API _ AdUnits', function () {
                 }
             ]
     }];
-
+    $$PREBID_GLOBAL$$.adUnits = [];
```

Comment:
if you look below this line you'll see:
https://github.com/prebid/Prebid.js/blob/master/test/spec/adUnits_spec.js#L49
Which invokes:
https://github.com/prebid/Prebid.js/blob/master/test/helpers/pbjs-test-only.js#L8
So there is no need to do this. I'm not sure why it's hidden away like this, maybe to discourage use of mocking with prebid.js internals.